src/App/Pages/LoginTwoFactorPage.cs

```diff
@@ -232,17 +232,22 @@ private void Init()
 
                 var image = new CachedImage
                 {
-                    Source = "yubikey",
+                    Source = "yubikey.png",
                     VerticalOptions = LayoutOptions.Start,
                     HorizontalOptions = LayoutOptions.Center,
                     WidthRequest = 266,
                     HeightRequest = 160,
                     Margin = new Thickness(0, 0, 0, 25)
                 };
 
+                TokenCell = new FormEntryCell(AppResources.YubiKeyTextEnter, imageSource: "lock");
```

Comment:
`AppResources.YubiKeyTextEnter` This text can go away to be consistent with the browser extension. It is blank.

Reply:
Also, is this field auto-focused?

Reply:
Ok, fixed